src/tests/unit/jest.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     roots: [currentDir],
     collectCoverage: true,
     collectCoverageFrom: ['<rootDir>/src/**/*.{ts,tsx}', '!<rootDir>/src/tests/**/*', '!<rootDir>/src/**/*.d.ts'],
-    coverageReporters: ['json', 'lcov', 'text', 'cobertura'],
+    coverageReporters: ['json', 'text', 'cobertura', 'html'],
```

Comment:
lcov generates the html report: ./accessibility-insights-web/test-results/unit/coverage/lcov-report/index.html